androidtv/basetv/basetv.py

```diff
@@ -128,6 +128,9 @@ def _cmd_audio_state(self):
         if constants.CUSTOM_AUDIO_STATE in self._custom_commands:
             return self._custom_commands[constants.CUSTOM_AUDIO_STATE]
 
+        # Is this a Android 11 Android TV?
```

Comment:
```suggestion
        # Is this an Android 11 device?
```

Please update the other occurrences, as well.